tests/sentry/ratelimits/utils/test_get_rate_limit_value.py

```diff
@@ -66,22 +64,12 @@ class Mixin:
             rate_limits = {"GET": {RateLimitCategory.IP: RateLimit(2, 4)}}
 
         class ChildEndpoint(ParentEndpoint, Mixin):
-            rate_limits = {"GET": {}}
+            pass
```

Comment:
this is changed behaviour. We no longer do the runtime-merging of inherited rate limit configs